### DISCUSSION TITLE
Duplicate r2modman upload by "ziyong?"

### DISCUSSION THREAD

**Discussion by iskawhiskers** [ebkr/r2modmanPlus#672] [Q&A]
https://thunderstore.io/package/ziyong/r2modman/

Appears to be a duplicate of the entire Thunderstore blurb for r2modman, including linking back to the github.  I don't know how to report mods on Thunderstore (if such a function exists).  There also seem to be no forks/branches under the name "ziyong"

**Comment by harbingerofme** [comment]
Taken care of
